Changes.md

```diff
@@ -2,6 +2,57 @@
 
 ## H2O
 
+### zz_Kurka (3.40.0.1) - 2/8/2023
```

Comment:
Here we can use just ``Kurka`` imho. `zz_` serves only to preserve the alphabetical sequence of major releases.